packages/rest/src/index.ts

```diff
@@ -168,14 +263,41 @@ class Rest {
                             query
                         )
 
-                        const [entities, total] = await manager.findAndCount(
+                        const populateValues = Object.values(
+                            findOptions.populate as any
+                        ).map(item => Object.values(item as any))[0]
+
+                        const res = this.transformToInfoObject(
```

Comment:
Refactor this entire thing our of the handlers, so you can just call one function instead of calling multiple functions just to get the `infoObj`